packages/components/src/panel/body.native.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * External dependencies
+ */
+import { View } from 'react-native';
+
+/**
+ * WordPress dependencies
+ */
+import { Component } from '@wordpress/element';
+
+export class PanelBody extends Component {
+	constructor( props ) {
+		super( ...arguments );
+		this.state = {
+			opened: props.initialOpen === undefined ? true : props.initialOpen,
+		};
+		this.toggle = this.toggle.bind( this );
+	}
+
+	toggle( event ) {
+		event.preventDefault();
+		if ( this.props.opened === undefined ) {
+			this.setState( ( state ) => ( {
+				opened: ! state.opened,
+			} ) );
+		}
+
+		if ( this.props.onToggle ) {
+			this.props.onToggle();
+		}
+	}
+
+	render() {
+		const { children, opened } = this.props;
+		const isOpened = opened === undefined ? this.state.opened : opened;
+		// const styles = [nativeStyles.components-panel__body, nativeStyles[className], isOpened ? nativeStyles.isOpened : {} ];
+		const styles = {}; //{backgroundColor: 'red'}
+
+		return (
+
+			<View style={ styles } >
+				{ /* { title && <Text>{title}</Text>} */ }
+				{ isOpened && children }
+			</View>
+		);
+	}
+}
+
+export default PanelBody;
```

Comment:
prepare simple PanelBody implementation - TBD what do do here next

Reply:
Ok, I will exclude PanelBody component and just render View instead.

Maybe if block has more sections the `TabView` after show `BottomSheet` components will be more user-friendly to show more settings sections on mobile ?

Reply:
sorry if I wasn't clear before, I think we should keep the PanelBody but just render a simple View in it for now.

Reply:
Done.